### PR TITLE
i18n(zh-cn): Update toc title

### PR DESCRIPTION
**Pull request by liruifengv**
#### Description

- improve translation follow https://github.com/withastro/docs/pull/5036

### CODE REVIEW

**Review by delucis — COMMENTED** [review]
Thanks @liruifengv! Could you also add a changeset to this PR? You should be able to use the “Click here if you're a maintainer who wants to add a changeset to this PR” link in the Changeset comment or run `pnpm changeset` to use the CLI locally.

Because these translations are shipped as part of Starlight, the changeset makes sure the change here is mentioned in the changelog for the next release.

**Comment by liruifengv** [comment]
> Thanks @liruifengv! Could you also add a changeset to this PR? You should be able to use the “Click here if you're a maintainer who wants to add a changeset to this PR” link in the Changeset comment or run `pnpm changeset` to use the CLI locally.
> 
> 
> 
> Because these translations are shipped as part of Starlight, the changeset makes sure the change here is mentioned in the changelog for the next release.

Done

File: .changeset/big-pots-destroy.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"@astrojs/starlight": patch
+---
+
+i18n(zh-cn): Improve the toc title translation
```
Comment (delucis) [inline .changeset/big-pots-destroy.md:5]: Thank you!

Suggesting a small change to match the style Sarah has recommended to us for changelogs:

```suggestion
Improves the table of contents title translation in Simplified Chinese
```

(Basic rules are: start with a third-person verb like “improves”, “fixes”, “adds” etc. and use a full sentence. More details: https://contribute.docs.astro.build/guides/new-feature-docs/#changesets)

**Review by delucis — APPROVED** [review]
Thanks again @liruifengv! LGTM 🚀

**Pull request merged**